python/ray/ml/__init__.py

```diff
@@ -0,0 +1,5 @@
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.config import RunConfig, ScalingConfig
+from ray.ml.preprocessor import Preprocessor
+
+__all__ = ["Checkpoint", "Preprocessor", "RunConfig", "ScalingConfig"]
```

Comment:
Just a minor point, but I personally like having users do fully qualified package imports instead of putting things into the ray.ml parent package.

Reply:
I think we may want to discuss this sometime - I think for the main concepts we can move them to the parent package as they will be used often. We shouldn't do this e.g. for trainer implementations